### PR TITLE
MaxPool versioning in quantization tools.

### PR DESCRIPTION
**Pull request by zhanghuanrong**
Ensure maxpooling quantization works correctly before and after opset12

### CODE REVIEW

File: onnxruntime/python/tools/quantization/onnx_quantizer.py [hunk 1]
```diff
@@ -132,8 +132,10 @@ def check_opset_version(self):
                 .format(opset_version))
             self.model.model.opset_import.remove(ai_onnx_domain[0])
             self.model.model.opset_import.extend([onnx.helper.make_opsetid("", 11)])
+            opset_version = 11
         
         self.fuse_dynamic_quant = True
+        return opset_version
```
Comment (RandySheriffH) [inline onnxruntime/python/tools/quantization/onnx_quantizer.py:138]: >opset_version [](start = 15, length = 13)

The function is not supposed to return a value or not?
Line 127 has a simple return.

Comment (zhanghuanrong) [reply onnxruntime/python/tools/quantization/onnx_quantizer.py:138]: need return the value to make outside logic looks clear.

Comment (RandySheriffH) [reply onnxruntime/python/tools/quantization/onnx_quantizer.py:138]: Cool, then how about let 127 also return something to make the function more consistent?

---
In reply to: [489639052](https://github.com/microsoft/onnxruntime/pull/5194#discussion_r489639052) [](ancestors = 489639052)

Comment (zhanghuanrong) [reply onnxruntime/python/tools/quantization/onnx_quantizer.py:138]: fixed

**Comment by RandySheriffH** [comment]
>         if opset_version == 10:

All good but could it be a bit more sorted like:

if 10 == opset_verison:
    print(...)
else:
   if opset_version < 10:
       ...
   self.fuse_dynamic_quant = True
return opset_version
  

---
Refers to: onnxruntime/python/tools/quantization/onnx_quantizer.py:123 in 7ac2a68. [](commit_id = 7ac2a6800c378cb361a893602ccde692a28e2e58, deletion_comment = False)

**Review by RandySheriffH — APPROVED** [review]
:shipit:

**Comment by zhanghuanrong** [comment]
> > ```
> >     if opset_version == 10:
> > ```
> 
> All good but could it be a bit more sorted like:
> 
> if 10 == opset_verison:
> print(...)
> else:
> if opset_version < 10:
> ...
> self.fuse_dynamic_quant = True
> return opset_version
> 
> Refers to: onnxruntime/python/tools/quantization/onnx_quantizer.py:123 in [7ac2a68](https://github.com/microsoft/onnxruntime/commit/7ac2a6800c378cb361a893602ccde692a28e2e58). [](commit_id = [7ac2a68](https://github.com/microsoft/onnxruntime/commit/7ac2a6800c378cb361a893602ccde692a28e2e58), deletion_comment = False

**Pull request merged**